internal/fnruntime/container.go

```diff
@@ -170,8 +174,21 @@ func (f *ContainerFn) prepareImage() error {
 	}
 	if strings.Contains(string(output), strings.Split(f.Image, ":")[0]) {
```

Comment:
This only check the existence of image name without tag. I am thinking using [`docker image inspect`](https://docs.docker.com/engine/reference/commandline/image_inspect/).